modules/cert-manager-acme-dns01-explicit-aws.adoc

```diff
@@ -22,6 +22,11 @@ You can use Amazon Route 53 with explicit credentials in an {product-title} clus
 . Optional: Override the nameserver settings for the DNS-01 self check.
 +
 This step is required only when the target public-hosted zone overlaps with the cluster's default private-hosted zone.
++
+[IMPORTANT]
+====
+The override procedure is applicable for all DNS-01 self-checks.
+====
```

Comment:
It's great to add this kind of message for user to refer. BTW, to make the document more detailed, do we need to reference this part in other modules?

Reply:
It does make sense to give a reference in other modules. I will add a note or additional info in other modules.